FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1033,6 +1036,355 @@ paths:
       security:
       - OAuth2: [fraihmwork.component.write, fraihmwork.component.admin]
 
+  /monitor/v1/relationship/definition:
+    get:
+      tags:
+      - Relationships
+      - Monitor
+      summary: Retrieve all of the relationship definitions active in FRAIHMWORK
+      operationId: findAllRelationshipDefinitions
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RelationshipDefinition'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        403:
+          description: Forbidden - client is not authorized to read components
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        400:
+          description: Bad Request
```

Comment:
Was going to justify by saying that there are a lot of reasons to have a 400 outside of a bad parameter or payload, but probably easier to remove rather than having to add 400s to every endpoint for completeness 🤮

Reply:
Did not add optional param for name, as names are unique. Generally the query parameters help to filter, not to identify. Instead, added another operation below to query a specific relationship definition